src/CLIFramework/IO/NullStty.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace CLIFramework\IO;
+
+class NullStty implements Stty
```

Comment:
The purpose of the class is to avoid checking whether stty is supported on an environment or not every time. We choose the implementation of `Stty` in `ServiceContainer`, so classes which depend on `Stty` and use `ServiceContainer` to get `Stty` instance don't have to care about that invocations of `Stty` methods would raise an exception.
Do you think that is too much abstraction or meaningless?